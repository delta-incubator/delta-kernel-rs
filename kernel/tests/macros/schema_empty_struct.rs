extern crate delta_kernel_derive;
use delta_kernel_derive::Schema;

mod actions;
mod schema;

#[derive(Schema)]
pub struct NoFields;

fn main() {}
