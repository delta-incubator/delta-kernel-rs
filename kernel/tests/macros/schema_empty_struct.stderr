error: this derive macro only works on structs with named fields
 --> tests/macros/schema_empty_struct.rs:7:10
  |
7 | #[derive(Schema)]
  |          ^^^^^^
  |
  = note: this error originates in the derive macro `Schema` (in Nightly builds, run with -Z macro-backtrace for more info)
